quic/s2n-quic-qns/src/perf.rs

```diff
@@ -289,7 +289,7 @@ pub struct Counters {
 
 impl Counters {
     pub fn print_header(&self) {
-        eprintln!(
+        println!(
```

Comment:
Doing a `println` instead of a `eprintln` makes it easier to pipe the output to a tsv file for later use